src/Farmer/Arm/ContainerService.fs

```diff
@@ -0,0 +1,88 @@
+[<AutoOpen>]
+module Farmer.Arm.ContainerService
+
+open Farmer
+open Farmer.CoreTypes
+open Farmer.Vm
+
+let managedClusters = ResourceType "Microsoft.ContainerService/managedClusters"
+
+type AgentPoolMode = System | User
+
+type ManagedCluster =
```

Comment:
Looks good - there's a lot of bells and whistles for AKS but this gets the big stuff